modules/ROOT/pages/sync/authentication.adoc

```diff
@@ -0,0 +1,25 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Authentication for {sync-service} service
+
+== Using the {sync-service} SDK with {keycloak-service} SDK
```

Comment:
{sync-service} Authentication ?

Reply:
+1 we should describe the benefit to the user not the technologies, especially in headings